asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py

```diff
@@ -153,7 +158,7 @@ def large_scale_docking_workflow(inputs: LargeScaleDockingInputs):
             dask_cluster.scale(inputs.dask_cluster_n_workers)
 
         dask_client = Client(dask_cluster)
-        # dask_client.forward_logging() distributed vs dask_cuda versioning issue, see # #669
+        dask_client.forward_logging()  # distributed vs dask_cuda versioning issue, see # #669
```

Comment:
is this no longer an issue?

Reply:
Yes good spotting, has been fixed upstream.